src/model/json-schema.json

```diff
@@ -3721,6 +3707,9 @@
                 "isLive": {
                     "type": "boolean"
                 },
+                "isPreview": {
+                    "type": "boolean"
+                },
```

Comment:
This seems strange that it appears in the schema but is not changed on the TS model?

Reply:
good point! will revert